tempodb/backend/s3/config.go

```diff
@@ -10,7 +10,7 @@ type Config struct {
 	Bucket            string         `yaml:"bucket"`
 	Endpoint          string         `yaml:"endpoint"`
 	Region            string         `yaml:"region"`
-	AccessKey         flagext.Secret `yaml:"access_key"`
+	AccessKey         string         `yaml:"access_key"`
```

Comment:
I believe this change should be mentioned in the changelog as its a user facing change. The YAML marshaller will no longer print `AccessKey` as `********`

Reply:
Good point added a changelog entry